src/theme/core.css

```diff
@@ -0,0 +1,10 @@
+app {
+    font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
+    color: #000000;
+}
+
+ion-button {
+    height: 0.3in;
+    width: 1in;
+    --background-color: --ion-color-medium-contrast;
+}
```

Comment:
Is this global design necessary? I think you can just do this in component level?

Reply:
It's intended for generic designs like colors and fonts. Removed height and width. Positions and size will be component side.